### PR TITLE
 Added RRDGLCD support for the new Watch Screen.

### PR DESCRIPTION
**Pull request by garikdan**
- Added graphics functions.
pixel
drawLine
drawHLine
drawVLine
drawBox
- Added function to set the cursor position.
setCursorPX and setCursor.
- Improved functions.
displayChar
displayString
renderChar
- Changed deg symbol.

### CODE REVIEW

File: src/modules/utils/panel/panels/rrdglcd/RrdGlcd.cpp [hunk 1]
```diff
@@ -297,6 +316,68 @@ void RrdGlcd::renderGlyph(int xp, int yp, const uint8_t *g, int pixelWidth, int
...
+    if (y < HEIGHT && x < WIDTH)
+  {
+    unsigned char mask = 0x80 >> (x % 8);
+    unsigned char *byte = &fb[y * (WIDTH/8) + (x/8)];
+    if ( color == 0 )
+        *byte &= ~mask; // clear pixel
+    else
+        *byte |= mask; // set pixel
+	    // Change the dirty rectangle to account for a pixel being dirty (we assume it was changed)
+    if (startRow > y) { startRow = y; }
```
Comment (wolfmanjm) [inline src/modules/utils/panel/panels/rrdglcd/RrdGlcd.cpp]: These next 4 variables appear to never be used for anything, so why waste RAM by defining them?

**Review by wolfmanjm — CHANGES_REQUESTED** [review]

**Comment by wolfmanjm** [comment]
Thank you for doing this.
It would be nice if you could fix the indentation to comply with the smoothie standard though. There are some areas where indentation is totally incorrect.
- Thank you

File: src/modules/utils/panel/panels/rrdglcd/RrdGlcd.cpp (hunk 1 above)
Comment (garikdan) [reply src/modules/utils/panel/panels/rrdglcd/RrdGlcd.cpp]: Removed unused variables.

**Comment by wolfmanjm** [comment]
excellent thank you. i'll test this today.

**Comment by garikdan** [comment]
**Now the conclusion as it should be:**

img_20190228_205937
img_20190228_205853
img_20190228_211029

**Comment by arthurwolf** [comment]
Hey there, why was this deleted ? I think it's worth keeping open for people to test ( and thanks for the code btw ).

**Pull request closed without merging**